hubblestack_nova_profiles/cis/centos-7-level-1-scored-v2-1-0.yaml

```diff
@@ -512,6 +512,16 @@ grep:
             pattern: Ciphers
             tag: CIS-5.2.11
       description: Ensure only approved ciphers are used
+    sshd_approved_macs:
+      data:
+        CentOS Linux-7:
+        - /etc/ssh/sshd_config:
+            match_output: 'MACs hmac-sha2-512-etm@openssh.com,hmac-sha2-256-etm@
+openssh.com,umac-128-etm@openssh.com,hmac-sha2-512,hmac-sha2-256,umac-128@openss
+h.com'
```

Comment:
Looks like hard line wrapping was turned on. Can you push this string onto a single line? Otherwise YAML won't be happy.